### PR TITLE
[CDAP-4600] change copyright date to just get current year

### PR DESCRIPTION
**Pull request by elfenheart**
screen shot 2016-01-13 at 4 48 53 pm

### CODE REVIEW

File: cdap-ui/app/main.js [hunk 1]
```diff
@@ -310,6 +310,8 @@ angular
       getVersion();
     }
 
+    $scope.copyrightYear = new Date().getFullYear();
```
Comment (bdmogal) [inline cdap-ui/app/main.js]: shouldn't this be ``'2015-' + new Date().getFullYear();``

Comment (elfenheart) [inline cdap-ui/app/main.js]: When we had to change this last year, it was said that we just need to include the latest year

Comment (bdmogal) [inline cdap-ui/app/main.js]: that's probably because 2015 was the first year of the new UI? @awholegunch can you confirm?

Comment (elfenheart) [inline cdap-ui/app/main.js]: Confirmed from Jon that all we need is the current year

**Comment by bdmogal** [comment]
:+1:

**Pull request merged**